### PR TITLE
virsh_create: sleep a while to make sure guest is destroyed

### PR DESCRIPTION
**Pull request by Chenditang**
virsh.create "console_autodestroy_paused" case may return fail：
ERROR| TestFail: Guest still exist with options --console --autodestroy --paused
so delay to check vm status is necessary.

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_create.py [hunk 1]
```diff
@@ -70,6 +70,8 @@ def create_autodestroy_check(vm):
             """
             check if guest will disappear with --autodestroy
             """
+            # sleep to make sure guest is destroyed
+            time.sleep(2)
```
Comment (PandaWei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_create.py]: Indeed, 
It's simplest way to fix the problem by sleeping a while.
Maybe 2 seconds is enough for your machine.
But it doesn't work well for others.
so i advice you that check if VM exists once in a while and last one minute or more.
You can refer to utils_misc.wait_for(),
There are a lot of example for it at tp-libvirt.

Comment (Chenditang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_create.py]: @PandaWei 
Updated, Thank you very much!

**Comment by Chenditang** [comment]
The test result on libvirt-1.1.3.9-1.fc20.x86_64

\# ./run -k --no-downloads -t libvirt --tests virsh.create
SETUP: PASS (0.20 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /root/autotest/client/tests/virt/logs/run-2015-07-23-17.07.04/debug.log
TESTS: 9
(1/9) type_specific.io-github-autotest-libvirt.virsh.create.none: PASS (23.49 s)
(2/9) type_specific.io-github-autotest-libvirt.virsh.create.console: PASS (21.96 s)
(3/9) type_specific.io-github-autotest-libvirt.virsh.create.console_autodestroy: PASS (18.46 s)
(4/9) type_specific.io-github-autotest-libvirt.virsh.create.console_paused: PASS (22.52 s)
(5/9) type_specific.io-github-autotest-libvirt.virsh.create.console_autodestroy_paused: PASS (19.47 s)
(6/9) type_specific.io-github-autotest-libvirt.virsh.create.autodestroy: PASS (3.36 s)
(7/9) type_specific.io-github-autotest-libvirt.virsh.create.autodestroy_paused: PASS (3.45 s)
(8/9) type_specific.io-github-autotest-libvirt.virsh.create.paused: PASS (23.50 s)
(9/9) type_specific.io-github-autotest-libvirt.virsh.create.negtive_readonly: PASS (0.78 s)
TOTAL TIME: 137.04 s (02:17)
TESTS PASSED: 9
TESTS FAILED: 0
SUCCESS RATE: 100.00 %

**Comment by PandaWei** [comment]
ACK and Merge

**Pull request merged**